cmd/kubeapps-apis/proto/kubeappsapis/plugins/resources/v1alpha1/resources.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+package kubeappsapis.plugins.resources.v1alpha1;
+option go_package = "github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/plugins/resources/v1alpha1";
+
+import "kubeappsapis/core/packages/v1alpha1/packages.proto";
+
+import "google/api/annotations.proto";
+import "google/protobuf/any.proto";
+
+service ResourcesService {
+    rpc GetResources(GetResourcesRequest) returns (stream GetResourcesResponse) {
+        option (google.api.http) = {
+            get: "/plugins/resources/v1alpha1/{installed_package_ref.plugin.name}/{installed_package_ref.plugin.version}/c/{installed_package_ref.context.cluster}/ns/{installed_package_ref.context.namespace}/{installed_package_ref.identifier}"
+        };
+    }
+}
+
```

Comment:
API docs nitpick hat: I'd rather use this construct to generate both the title and description of each field, just like in [here](https://github.com/kubeapps/kubeapps/blob/v2.4.1/cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto#L281-L295):

``` 
// title
//
// long description
whatever whatever whatever whatever 
``` 

Not a big deal though, totally up to you

Reply:
Will do, thanks.